src/utils/supports.js

```diff
@@ -39,10 +48,17 @@ function supports(feature) {
 
     const range = features[feature];
     if (!range) return false;
-    return (supportsCache[key] = satisfies(version, range, {
-      includePrerelease: true,
-      loose: true,
-    }));
+
+    // '@mastodon/blah' => 'mastodon'
+    const featureSoftware = feature.match(/^@([a-z]+)\//)[1];
```

Comment:
This needs to be like `(feature.match(/^@([a-z]+)\//) || [,false])[1]` because if there's no match, `match` will return `null`.

I know my code above looks fancy (it's in other files) so feel free to write it in a more understandable way.

Reply:
I thought I could rely on `supports()` only being called with strings that match the `@software/feature-name` pattern; currently, this is the case:

```
$ git grep supports\(
src/components/account-info.jsx:                {supports('@mastodon/profile-private-note') && (
src/components/account-info.jsx:              supports('@mastodon/profile-edit') && (
src/components/compose.jsx:                if (editStatus && supports('@mastodon/edit-media-attributes')) {
src/components/compose.jsx:                {(supports('@pleroma/local-visibility-post') ||
src/components/compose.jsx:                  supports('@akkoma/local-visibility-post')) && (
src/components/compose.jsx:  const supportsEdit = supports('@mastodon/edit-media-attributes');
... etc., all of the calls are like this
```

So it seems like a call which did not match that pattern would likely be a bug. Do you want me to check for that here anyway?